cogs/respects/respects.py

```diff
@@ -178,6 +181,33 @@ async def checkLastRespect(self, ctx):
 
         if not chData[KEY_MSG]:
             return False
+        else:
+            oldRespect = None
+            try:
+                oldRespect = await ctx.channel.fetch_message(chData[KEY_MSG])
+            except discord.NotFound:
+                self.logger.debug("Could not find the old respect")
+                return False
+            except discord.Forbidden:
+                self.logger.debug("Did not have permissions to get the old respect")
+                await ctx.send(
+                    'I currently cannot get messages, please give me "Manage'
+                    ' Message" permissions to allow this feature to work!'
+                )
+            except discord.HTTPException:
+                self.logger.debug("Could not retrieve the old respect")
```

Comment:
Since this is one of the more generic exceptions and we shouldn't hit this normally, what are your thoughts on setting the log level to `error` and printing out the traceback?
```suggestion
            except discord.HTTPException:
                self.logger.error("Could not retrieve the old respect", exc_info=True)
```

Reply:
I agree.
HTTP requests should not fail in normal operation. Might signify abnormalities.